### PR TITLE
changes the master branch to main and updates references to master with main in the codebase

### PR DESCRIPTION
**Pull request by srhrshr**

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.. RSMTool documentation master file, created by
+.. RSMTool documentation main file, created by
```
Comment (desilinguist) [inline doc/index.rst:1]: We don't really need to change it here since that's coming from a Sphinx template.

Comment (srhrshr) [reply doc/index.rst]: My bad! Reverted this change.

**Review by desilinguist — APPROVED** [review]
LGTM. @aloukina can you please also take a look?

**Review by aloukina — APPROVED** [review]
Looks good to me - thank you!

**Pull request merged**